samples/CentOS/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM centos:7
```

Comment:
Missing standard license header.

Reply:
Would appreciate if you can share a link to the standard license header that needs to be used. Is it mandatory for Dockerfiles and docker-compose files as well?